modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/RichText/RichText.es.js

```diff
@@ -271,7 +270,10 @@ const RichText = ({
 						<CKEditor5ClassicEditor
 							className="w-100"
 							config={{
-								...advancedClassicEditorConfig,
+								filebrowserImageBrowseUrl:
+									editorConfig.filebrowserImageBrowseUrl,
+								itemSelectorEventName: `${name}selectItem`,
+								preset: 'advanced',
```

Comment:
Instead of exporting `advancedClassicEditorConfig`, the idea is that presets are set with the keyword. This enables easy setting of presets in `EditorConfigContributor`.

Reply:
this takes the pattern from the CK4 toolbar config mecanism doesn't it?